iscsi/targetd/provisioner/iscsi-provisioner.go

```diff
@@ -192,7 +206,7 @@ func (p *iscsiProvisioner) createVolume(options controller.VolumeOptions) (vol s
 	}
 	log.Debugln("created volume name, size, pool: ", vol, size, pool)
 	for _, initiator := range initiators {
-		log.Debugln("exporting volume name, lun, pool, initiatir: ", vol, lun, pool, initiator)
+		log.Debugln("exporting volume name, lun, pool, initiator: ", vol, lun, pool, initiator)
```

Comment:
why dont you use `glog` instead of `log`?

Reply:
I am not familiar with glog, is it a requirement? It wasn't last time I send a pull request.
the log variable is actually an instance of logrus a popular and feature rich log library in go

Reply:
@raffaelespazzoli `glog` is the preferred logging mechanism in kube land and very simple to use please check https://github.com/kubernetes/community/blob/master/contributors/devel/logging.md . On whether its mandatory or not, I would invite @wongma7 @jsafrane @rootfs

Reply:
@humblec and other please let me  know.
I would like to complete this PR so I can work on CHAP authentication.
I suggest to postpone any refactoring such as this log library refractory, if it turns out to be necessary.